brainstormforce/ultimate-addons-for-gutenberg
# Fix: Team - Updated default value on frontend.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/569fdb2b-98be-4a62-b699-4b0780dabf04?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes
- Updated default value on the frontend.

### Screenshots


### Types of changes




- Updation

### How has this been tested?

- Drag and drop Team block.
- Check if the default value applied on the frontend same as mentioned in the editor setting.

### Checklist:
